pandas/tests/tseries/offsets/test_offsets_properties.py

```diff
@@ -100,7 +100,7 @@ def test_on_offset_implementations(dt, offset):
     #   (dt + offset) - offset == dt
     try:
         compare = (dt + offset) - offset
-    except pytz.NonExistentTimeError:
+    except (pytz.NonExistentTimeError, pytz.AmbiguousTimeError):
         # dt + offset does not exist, assume(False) to indicate
```

Comment:
Could you just expand the comment here to note that the above can also produce an ambiguous time as well?

Reply:
Def. Does this work?

Reply:
Yes, thanks.